src/sympc/module/__init__.py

```diff
@@ -7,23 +7,26 @@
 # stdlib
 from collections import OrderedDict
 import copy
-from typing import Any
-from typing import Dict
 
 # third party
 import syft as sy
 import torch
 
 import sympc.module as sympc_module
+from sympc.session import Session
 
 from .nn import Conv2d
 from .nn import Linear
 
-MAP_TORCH_TO_SYMPC = {torch.nn.Linear: Linear, torch.nn.Conv2d: Conv2d}
+MAP_TORCH_TO_SYMPC = {
+    "Linear": Linear,
+    "Conv2d": Conv2d,
+    "Conv2dPointer": Conv2d,
+    "LinearPointer": Linear,
+}
```

Comment:
nit: maybe add dynamically the Pointer versions?

Reply:
Good idea